packages/lakka/lakka_tools/gpicase_safeshutdown/scripts/safeshutdown_gpi.py

```diff
@@ -5,18 +5,27 @@
 
 from gpiozero import Button, LED
 import os
+import io
 from signal import pause
 
 powerPin = 26
+powerPinPullUp = True
 powerenPin = 27
 hold = 1
 power = LED(powerenPin)
 power.on()
 
+os_release_file = io.open("/etc/os-release")
+device = list(filter(lambda x: x.find('LIBREELEC_DEVICE') == 0, os_release_file.readlines() ))[0][18:-2]
+os_release_file.close()
+
+if device == "RPi4-GPICase2":
+    powerPinPullUp = False
+
```

Comment:
[18:-2] is reducing `LIBREELEC_DEVICE="RPi4-GPICase2"` to `RPi4-GPICase`. Below should do what you want (double check, the hour is late):

```
with open('/etc/os-release', 'r') as data:
    content = data.read()
device = list(filter(lambda x: x.find('LIBREELEC_DEVICE') == 0, content.splitlines()))[0][18:-1]

powerPinPullUp = False if device == 'RPi4-GPICase2' else True
```

Reply:
Tested and working, thanks!